### ISSUE TITLE
Anpassung an VIS-2

### ISSUE THREAD

**Issue by Blockmove** [Pittini/iobroker-heatingcontrol-vis#27]
Ist eine Version für die VIS-2 geplant?

**Comment by Pittini** [comment]
Geplant nein, möglich ja, aber eher mittelfristig. Läuft die aktuelle Version denn gar nicht auf der 2?

**Comment by Blockmove** [comment]
Sagen wir mal so: Heating-Control ist bedienbar.
Es werden aber schon viele Anpassungen notwendig.
Aber "mittelfristig" ist aus meiner Sicht ok.

**Comment by detafun** [comment]
Wollte auch gerade mit VIS-2 anfangen und leider funktioniert dein Tolles Heating-Controll noch noch nicht. Wäre klasse, wenn es funktionieren würde.

**Comment by Blockmove** [comment]
> Wollte auch gerade mit VIS-2 anfangen und leider funktioniert dein Tolles Heating-Controll noch noch nicht. Wäre klasse, wenn es funktionieren würde.

Ich hab jetzt einige Zeit in das Testen der VIS-2 gesteckt. Im aktuellen Stand sind noch viele Unzulänglichkeiten bei den Widgets vorhanden. Selbst bei der Basic-Variante der Heating-Control-Visu hakt es noch an etlichen Stellen.

**Comment by DJ3vil** [comment]
Hallo, 
gibts es hier Fortschritte ? Ich kann leider die VIS nicht in VIS 2 importiere, bekomme immer nur die Vis Auswahl meiner Projekte nach klicken auf HeatingControl

**Comment by Pittini** [comment]
> Hallo, gibts es hier Fortschritte ? Ich kann leider die VIS nicht in VIS 2 importiere, bekomme immer nur die Vis Auswahl meiner Projekte nach klicken auf HeatingControl

S olange Vis2 noch im Beta Stadium ist, und soo vieles nicht klappt, werd ich da nix machen. Umstellung gibts dann, wenn ich für mich selber auf die 2 umsteige.

**Comment by Esmax666** [comment]
@Blockmove hast du etwas geschfat auf VIS2 etwas zu machen?

@Pittini jetzt ist Vis2 nicht mehr als beta :) ich hoffe das VIS2 besser läuft für heatingcontrol 

Danke für feedkback :)

**Comment by Blockmove** [comment]
@Esmax666 


> @Blockmove hast du etwas geschfat auf VIS2 etwas zu machen?
> 
Ich hab mich von VIS bzw VIS2 verabschiedet und bin auf webui als Visualisierung umgestiegen.

**Comment by Esmax666** [comment]
> @Esmax666
> 
> > @Blockmove hast du etwas geschfat auf VIS2 etwas zu machen?
> 
> Ich hab mich von VIS bzw VIS2 verabschiedet und bin auf webui als Visualisierung umgestiegen.

ach so warum? hast du ein template für webui? war das einfacher (schöner)

**Comment by Blockmove** [comment]
> > @Esmax666
> > > @Blockmove hast du etwas geschfat auf VIS2 etwas zu machen?
> > Ich hab mich von VIS bzw VIS2 verabschiedet und bin auf webui als Visualisierung umgestiegen.
> 
> ach so warum? hast du ein template für webui? war das einfacher (schöner)

webui verwendet normales HTML und CSS. Das ist mir persönlich einfach lieber.
Mit VIS und den anderen ioBroker-Visualisierungen bin ich über die Jahre nie so richtig warm geworden.

**Comment by aruttkamp** [comment]
Gibt es hier evtl. etwas neues ? @Pittini

**Comment by Pittini** [comment]
> Gibt es hier evtl. etwas neues ? @Pittini

Nope, nix neues, seh noch keine Notwendigkeit auf Vis2 umszusteigen.
